ilastik_scripts/ilastik_startup.py

```diff
@@ -138,9 +143,110 @@ def find_library(name):
     ctypes.util.find_library = find_library
 
 
+def check_parent_for_dir_recursive(path: pathlib.Path, dir_name: str) -> Optional[pathlib.Path]:
```

Comment:
Would it not be clearer if this function produced the target directory instead of the _parent_ of such directory? Then if the caller wants the parent, they cal always call `.parent` on the result. To that point, because this function produces the parent and not the target dir itself, the "ssl" component needs to be added back to the result to produce the correct paths to the certificates.

Reply:
In fact I think not having any function for this is clearer, since apparently it can be solved with a simple iteration :)